practices/software_development_kit.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: software-development-kit
+stages:
+  - development
+short_description: A SDK is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so.
+tags:
+  - sdk
+  - software development
+  - software engineering
+  - development
+  - developer experience
+keywords:
+  - product sprint
+  - software development
+  - software engineering
+  - product development
+  - debugging
+  - documentation
+  - libraries
+  - apis
+  - sdk
+---
+
+# Software Development Kit (SDK)
+
+**TL;DR**
+
+It is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so. You have what you need to get the job done, and what's included in your kit toolbox, varies from platform to platform.
+
+## What is an SDK
+
+A Software Development Kit (SDK) is the toolbox of the software development world. But instead of tools like screwdrivers and pliers, SDKs contain libraries and Application Programming Interfaces (APIs) that make it easier to create applications for specific software and/or hardware.
+
+It can also be thought of as a wrapper library developed by a service provider which provides an abstraction around the services or capabilites that the platform has or supports.
+
+![SDK](/files/software_development_kit.png)
+[CleverTap: What is an SDK? Everything You Need to Know](https://clevertap.com/blog/what-is-an-sdk/)
+
+### Why You Might Want an SDK
+
+An SDK can include a range of things, including libraries, documentation, code samples, processes, and guides that developers can use and integrate into their applicatios. SDKs are designed to be used for specific platforms or programming languages.
+
+All of this is done through the libraries included in each SDK. A library is essentially a piece of software that allows your application to communicate with the specific platform it is running on. Tools for debugging, compilation and relevant documentation are often included in SDKs, which makes it easy for developers to fix or extend their application.
```

Comment:
I have found similar text on https://www.freecodecamp.org/news/whats-an-sdk-software-development-kits-explained-in-plain-english/ (not in sources)

_All of this is done through the libraries included in each SDK. A library is essentially a piece of software that allows your application to communicate with the specific platform its running on._

_Tools for debugging and creating visual elements are often included in SDKs, making it easy for developers to fix their applications_